src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -613,3 +655,22 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
         }
     }
 }
+
+static class RandomExtensions
+{
+
+    public static string GetRandomSubstring(this Random random, string input)
+    {
+        var offset = random.Next(0, input.Length);
+        var length = random.Next(0, input.Length - offset + 1);
+        return input.Substring(offset, length);
+    }
+
+    public static bool NextBoolean(this Random random, double probability)
```

Comment:
Where is this used?

Reply:
I was working on randomized cancellation, however decided to postpone for a future PR. Forgot to remove this.